datalad_debian/build_package.py

```diff
@@ -183,4 +187,22 @@ def __call__(dsc, *, dataset=None, update_builder=False):
             result_renderer='disabled',
             return_type='generator',
             on_failure='ignore',
-        )
+        ):
+            # find the logfile
+            if r['status'] == 'ok' and r['action'] == 'add' \
+                    and r['type'] == 'file' \
+                    and r['refds'] == pkg_ds.pathobj.as_posix() \
+                    and r['path'].startswith(r['refds'] + "/logs"):
```

Comment:
Some of this could be simplified with 

https://github.com/psychoinformatics-de/datalad-debian/blob/4c6874434e106c50b870f75e286bfa9deffd2d2d/datalad_debian/utils.py#L3

Reply:
Hello 8 months ago in the past, I found this PR thanks to the health report. I applied this suggestion in [edfa0aa](https://github.com/psychoinformatics-de/datalad-debian/commit/edfa0aaf92828f68c67ba87bdbb68576b760932a)